labapi/src/main/java/com/microsoft/identity/internal/test/labapi/ApiClient.java

```diff
@@ -945,6 +947,7 @@ public void onResponse(Response response) throws IOException {
      * @throws ApiException If fail to serialize the request body object
      */
     public Call buildCall(String path, String method, List<Pair> queryParams, List<Pair> collectionQueryParams, Object body, Map<String, String> headerParams, Map<String, Object> formParams, String[] authNames, ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
+        authNames[0] = AUTH_TYPE;
```

Comment:
I think we should either,

1. remove authNames from buildCall and construct it here.
or
2. pass authNames, but define its value outside.

Reply:
Addressed with first approach